### DISCUSSION TITLE
Regarding Synopsis

### DISCUSSION THREAD

**Discussion by mehboobpatel** [github/india#818] [General]
Is it compulsory to submit the project synopsis and if yes can you please share the sample format 
Please answer
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Comment by ey-mentor1** [comment]
Hello, project synopsis is mandatory.
Unfortunately we do not share any sample synopsis.
Please read the description and fill it wisely.
The details filled in synopsis should match with what you share in your GitHub repository and project link

Thank you
